src/main/resources/view/SuggestionsWindowView.fxml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.control.ListView?>
+<?import javafx.scene.layout.VBox?>
+<?import javafx.geometry.Insets?>
+
+<?import javafx.scene.control.Label?>
+<VBox fx:id="suggestionsBox" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1"
+      styleClass="suggestions-Window" >
+    <Label fx:id="suggestionsText" VBox.vgrow="NEVER" prefWidth="Infinity" managed="false">
```

Comment:
Do you think it's better to call this `responseText` to be consistent with the model?

Reply:
Yes, this will be clearer. Noted.